mysql/mysqldump_test.go

```diff
@@ -0,0 +1,561 @@
+// // Copyright 2020 Google LLC
+// //
+// // Licensed under the Apache License, Version 2.0 (the "License");
+// // you may not use this file except in compliance with the License.
+// // You may obtain a copy of the License at
+// //
+// //      http://www.apache.org/licenses/LICENSE-2.0
+// //
+// // Unless required by applicable law or agreed to in writing, software
+// // distributed under the License is distributed on an "AS IS" BASIS,
+// // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// // See the License for the specific language governing permissions and
+// // limitations under the License.
+
+package mysql
+
+import (
+	"bufio"
+	"fmt"
+	"math/bits"
+	"strings"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/spanner"
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestProcessMySQLDump(t *testing.T) {
```

Comment:
I would suggest to split this giant test into smaller ones.

Reply:
Tried to follow same structure as `postgres/pgdump_test.go` but sure, I will split tests into smaller ones.